plugins/by-name/lspconfig/default.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  config,
+  options,
+  ...
+}:
+let
+  # cfg = config.plugins.lspconfig;
+  opts = options.plugins.lspconfig;
+  oldCfg = config.plugins.lsp;
+  oldOpts = options.plugins.lsp;
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "lspconfig";
+  packPathName = "nvim-lspconfig";
+  package = "nvim-lspconfig";
+
+  maintainers = with lib.maintainers; [
+    GaetanLepage
+    MattSturgeon
+    khaneliman
+    traxys
+  ];
```

Comment:
@HeitorAugustoLN you're currently marked as the `plugins.lsp` maintainer, since you did the `mkNeovimPlugin` migration. Do you want to be listed as a maintainer in the new plugin?

Also: does anyone currently in the list _not_ want to be? (@GaetanLepage, @khaneliman, @traxys)

Reply:
Sure! I will add myself after the merge